examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -100,6 +104,10 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 		this.uniformsGroup = {};
 
+		this.requires = {
```

Comment:
Maybe rename this to `builtins`? Then we can in future make it a Set and add full support for builtins.